e2e/benchmarks/task_trigger_test.go

```diff
@@ -0,0 +1,111 @@
+// +build e2e
```

Comment:
Not terribly sure why this file ended up rendering as a whole diff instead of [L25-27 like the other PR](https://github.com/hashicorp/consul-terraform-sync/pull/242/files#diff-768dbddc17b8942fcad403c0d38bb9657ca51c7922c11a47c2952d16ffca4726R25-R27).

Reply:
Oh... I'm seeing that this [file](https://github.com/hashicorp/consul-terraform-sync/pull/240) was merged into master after the release. Since my pr modified the file, looks like it got "resolved" here by committing the whole file 🤔

Reply:
Ah! that makes sense, thanks Lorna.